cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js

```diff
@@ -64,6 +66,10 @@ const selectOrganizationByName = (organizationName) => {
   ]);
 };
 
+const selectFromResultsList = (rowNumber = 0) => {
+  cy.do(Modal('Select Organization').find(MultiColumnListRow({ index: rowNumber })).click());
```

Comment:
please use the variable above
const organizationModal = Modal('Select Organization');

Reply:
Done